src/lib/merkle_mask/maskable_merkle_tree.ml

```diff
@@ -76,17 +84,23 @@ module Make (Inputs : Inputs_intf) = struct
     let open Graphviz in
     Uuid.Table.fold uuid_to_masks_table ~init:empty
       ~f:(fun ~key:uuid ~data:mask graph ->
-        let graph_with_mask = add_vertex graph mask in
-        Uuid.Table.find registered_masks uuid
-        |> Option.value_map ~default:graph_with_mask ~f:(fun children_masks ->
-               List.fold ~init:graph_with_mask children_masks
-                 ~f:(fun graph_with_mask_and_child ->
-                   add_edge graph_with_mask_and_child mask ) ) )
+        try_with "vertex" ~default:graph ~f:(fun () ->
```

Comment:
I don't fully understand the control flow here yet, but it would be very nice if dangling masks still showed up in the visualization with some easy way to tell them apart from attached masks.

Reply:
I made the update. I updated how a dangling mask looks like and how a attached mask looks like: https://ibb.co/vxNtqDG